static

```diff
@@ -0,0 +1 @@
+explore/target/scala-2.13/stage/static/
```

Comment:
This is a symlink.

It's used to have only one `static.json` configuration and for the app to work in both build situations: Heroku or remotely (GitHub).